test/recode/task/remove_parens_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Recode.Task.RemoveParensTest do
+  use RecodeCase
+
+  alias Recode.Task.RemoveParens
```

Comment:
We can fake the tests with the following code:
```suggestion
  use RecodeCase, async: true

  alias Recode.Task.RemoveParens

  @moduletag :tmp_dir

  setup context do
    File.write("#{context.tmp_dir}/.formatter.exs", """
    [
      inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
      locals_without_parens: [foo: 1, bar: 2]
    ]
    """)

    {:ok, context}
  end

  test "remove parens", %{tmp_dir: tmp_dir} do
    File.cd!(tmp_dir, fn ->
      code = """
      x = foo(bar(y))
      bar(y, x)
      """

      expected = """
      x = foo bar(y)
      bar y, x
      """

      code
      |> run_task(RemoveParens, autocorrect: true)
      |> assert_code(expected)
    end)
  end
```
The function `formatter_opts_for_file/1` starts the search always in the current directory.